CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Code review checklist (TODO: should be copy-pasted in PR template)
+
+- [ ] code should be formatted using `rustfmt` (:warning: this also applied to code in macro like `tokio::select!` and `lazy_static!`
+- [ ] code should be exempt of `clippy` lints (and function size should not exempt 50 lines)
```

Comment:
Why 50 lines ? Should we refactor every thing right now ?How to enforce that ? (I won't count your function's lines)

Reply:
@adrien-zinger discover that we can configure that into `clippy` :sparkles:

Reply:
Yes we can add that in clippy and fix warnings in follow ups. Moreover, can we speak about that in a meeting and finally merge that? :rofl: